lib/chartmogul/csv/csv_header.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+module ChartMogul
+  module CSV
+    module CSVHeader
+      def csv_file_headers
+        Struct.new(*fields).new(*headers)
```

Comment:
This creates a different anonymous class each time you call this :sweat: please move into a constant named as a class, eg: `Headers = Struct.new(*fields)`